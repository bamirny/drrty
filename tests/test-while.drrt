def int main(){
	int x;

	x = 1;
	while(x<5){
		prints("Hello");
		x = x+1;
	}
	return 0;
}