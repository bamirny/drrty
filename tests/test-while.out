<span>Hello</span><br>
<span>Hello</span><br>
<span>Hello</span><br>
<span>Hello</span><br>
